tracer/src/Datadog.Trace/Vendors/Newtonsoft.Json/Converters/BinaryConverter.cs

```diff
@@ -133,7 +133,7 @@ private static void EnsureReflectionObject(Type t)
             {
                 // current token is already at base64 string
                 // unable to call ReadAsBytes so do it the old fashion way
-                string encodedData = reader.Value!.ToString();
+                string encodedData = reader.Value!.ToString()!;
```

Comment:
each time I see a '!' it makes me thing of the '!important' in css. And then I'm sad :p

Reply:
I mean, it's an apt analogy 😂